python/paddle/nn/layer/loss.py

```diff
@@ -439,21 +436,25 @@ class HSigmoidLoss(Layer):
     Examples:
         .. code-block:: python
 
-            import paddle
-            paddle.set_device('cpu')
-
-            input = paddle.uniform([4, 3])
-            # [[0.56194401  -0.22450298  -0.10741806] # random
-            #  [0.36136317  0.23556745  0.88748658] # random
-            #  [0.18151939  0.80947340  -0.31078976] # random
-            #  [0.68886101  -0.14239830  -0.41297770]] # random
-            label = paddle.to_tensor([0, 1, 4, 5])
-            m = paddle.nn.HSigmoidLoss(3, 5)
-            out = m(input, label)
-            # [[2.42524505]
-            #  [1.74917245]
-            #  [3.14571381]
-            #  [2.34564662]]
+            >>> import paddle
+            >>> paddle.set_device('cpu')
+            >>> paddle.seed(2023)
+            >>> input = paddle.uniform([4, 3])
+            >>> print(input)
+            Tensor(shape=[4, 3], dtype=float32, place=Place(cpu), stop_gradient=True,
+            [[ 0.73167229,  0.04029441, -0.48078126],
+             [ 0.81050646, -0.15199822, -0.18717426],
+             [ 0.94041789,  0.48874724,  0.03570259],
+             [ 0.46585739,  0.95573163, -0.91368192]])
+            >>> label = paddle.to_tensor([0, 1, 4, 5])
+            >>> m = paddle.nn.HSigmoidLoss(3, 5)
```

Comment:
<img width="565" alt="image" src="https://github.com/PaddlePaddle/Paddle/assets/38436475/c0cce1b8-9938-4950-8030-dca078eaa536">

最后一个数据怎么差异这么大呢？而且只有这一个有差异

Reply:
```suggestion
            >>> m = paddle.nn.HSigmoidLoss(3, 6)
```
看了下，`num_classes`的参数输入改成`6`试试（如上），因为 label = paddle.to_tensor([0, 1, 4, 5]) 好像至少有6个class（0到5）

Reply:
孙师傅太厉害啦！